docs/AI-for-security/api/bulk-actions-anonymization-fields-api.asciidoc

```diff
@@ -0,0 +1,233 @@
+[[bulk-actions-anonymization-fields-api]]
+=== Bulk anonymization fields actions
+
+Applies a bulk action to multiple anonymization fields. The bulk action is applied to all anonymization fields that match the filter or to the list of anonymization fields by their IDs.
+
+[discrete]
+===== Request URL
+
+`POST  <kibana host>:<port>/api/security_ai_assistant/anonymization_fields/_bulk_action`
+
+[discrete]
+===== Request body
+
+A JSON object with the following properties:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+| `create` | <<bulk-create-anonymization-fields-schema, BulkCreateAction[]>> |  Array of anonymization fields objects to create. | No
+| `delete` | String[] |  Array of anonymization fields IDs to which a bulk action will be applied. | No
+| `update` | <<bulk-update-anonymization-fields-schema, BulkUpdateAction[]>> |  Array of anonymization fields objects with the fields to update. | No
+
+|==============================================
+
+
+[[bulk-create-anonymization-fields-schema]]
+[discrete]
+==== BulkCreateAction object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`field` |String |Field name in the document which will be send to LLM. |Yes
```

Comment:
```suggestion
|`field` |String |A field's name. |Yes
```
If I understand correctly, fields may not get sent to the LLM, depending on the `allowed` value. Updated suggestion here accordingly